docs/book/README.md

```diff
@@ -0,0 +1,26 @@
+# Kubernetes vSphere CSI Driver
+
+The [Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec/blob/master/spec.md) is a specification designed to enable persistent storage volume management on Container Orchestrators (COs) such as Kubernetes. This allows storage systems to integrate with containerized workloads running on Kubernetes. Using CSI, third-party storage providers (like VMware) can write and deploy plugins for storage systems in Kubernetes without a need to modify any core Kubernetes code.
+
+CSI allows volume plugins to be deployed (installed) on Kubernetes clusters as extensions. Once a CSI compatible volume driver is deployed on a Kubernetes cluster, users may use the CSI to provision, attach, mount and format the volumes exposed by the CSI driver. In the case of vSphere, the CSI driver is csi.vsphere.vmware.com.
+
+Please refer documentation for detail.
+
+## Documentation
```

Comment:
This section is exactly the same as SUMMARY.md?

Reply:
fwiw this is a common pattern. The `SUMMARY.md` file is used by gitbooks as a table of content which we probably want to also link in the home page.

Reply:
yes, SUMMARY.md file will be used to create side navigation bar, when html pages are generated using netlify.

README.md is just a landing page to help navigate mark down files, if some one is not using generated html site.

Reply:
@xing-yang @SandeepPissay @manojvs157 for your reference, this is what the site will look like with the structure added in this Pull Request.

https://5e8e250c0fc3db000638d75c--peaceful-haibt-c672b5.netlify.com/

Let's discuss if we want to change this.

Reply:
removed duplicate navigation for now. Once we decide on content for landing page, we will populate it in future.